webpack/scenes/ContentViews/ContentViewsCountHeader.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Spinner } from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { STATUS } from 'foremanReact/constants';
+import ContentViewIcon from './components/ContentViewIcon';
+
+const ContentViewsCountHeader = ({ response, status }) => {
+  const [componentCount, setComponentCount] = useState(<Spinner size="sm" />);
```

Comment:
Using JSX as state values seems very weird to me.  I think it's Patternfly table formatters etc. creating bad habits.  I'd stick to primitive values for state, and keep the JSX in the component's render method.  In other words:

1. ~~Store only the counts here in state - `useState(0)`~~
_update:_ I don't think state is needed here.  see below 
2. Have `ContentViewIcon` take a `loading` prop and render the `<Spinner>` there (this will also simplify your `useEffect` below.)

Reply:
I will change this. I was surprised when it worked this way at all..😄

Reply:
We need to add an API request here to show org wide content view counts. The current request is paged and changes upon changing pages or searching etc. 
This will probably need a new API endpoint and action to show the org-wide counts.